docs/pages/api.js

```diff
@@ -0,0 +1,755 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import fetch from 'cross-fetch';
+import AppFrame from 'docs/src/modules/components/AppFrame';
+import AppContainer from 'docs/src/modules/components/AppContainer';
+import Head from 'docs/src/modules/components/Head';
+import MarkdownElement, {
+  styles as markdownStyles,
+} from 'docs/src/modules/components/MarkdownElement';
+import clsx from 'clsx';
+import { makeStyles, styled } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Link from 'docs/src/modules/components/Link';
+import { pageToTitle } from 'docs/src/modules/utils/helpers';
+import EditPage from 'docs/src/modules/components/EditPage';
+import ScrollableTableOfContents from 'docs/src/modules/components/ScrollableTableOfContents';
+import Ad from 'docs/src/modules/components/Ad';
+import NoSsr from '@material-ui/core/NoSsr';
+
+const SOURCE_CODE_ROOT_URL = 'https://github.com/mui-org/material-ui/blob/master';
+const PATH_REPLACE_REGEX = /\\/g;
+const PATH_SEPARATOR = '/';
+
+function normalizePath(path) {
+  return path.replace(PATH_REPLACE_REGEX, PATH_SEPARATOR);
+}
+
+const InlineMarkdownElement = styled(MarkdownElement)({
+  display: 'inline',
+  '& p ': {
+    display: 'inline',
+  },
+});
+
+function ComponentImport(props) {
```

Comment:
Maybe we should be using the Markdown renderer here?

Reply:
why?

Reply:
It could allow us to use a single rendering logic. In the current form, what guarantee do we have that the class names or DOM structure will keep working in the future?